aaronArinder/mobile-disposition-tracker
# icon/splash update

This PR updates the icons/splashes for iOS and Android, away from the nativescript icons/splashes and to the welby robot.

#### testing
- pull down the changes, build the app locally
- make sure the icon and splash are updated, look good, and don't break anything

#21
